index.html

```diff
@@ -316,5 +328,122 @@ <h3 class="about-me__card__title">Skills</h3>
         </div>
       </div>
     </section>
+
+    <!-------CONTACT SECTION ------->
+    <section id="contact">
+      <div class="contact">
+        <div class="contact__container">
+          <h2>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
+
+          <form
+            class="contact__form"
+            action="https://formspree.io/f/xknaqawg"
+            method="POST"
+          >
+            <ul>
+              <li class="contact__form__element full-name">
+                <label class="label" for="full-name">Full name:</label>
+                <input
+                  type="text"
+                  id="full-name"
+                  name="full-name"
+                  placeholder="Full name"
+                  maxlength="30"
+                />
```

Comment:
This can be fixed with HTML/CSS and also it is not required to use JavaScript for this. If you are worried about the desktop version, please don't be. You'll soon get to know about it in upcoming milestone 😄